de_workloads/example_copy_job/terraform/date_pipeline.tf

```
@@ -0,0 +1,19 @@
+resource "fabric_data_pipeline" "example_ingest_pipeline" {
+  display_name = "example_ingest_pipeline"
+  description  = "Example data pipeline with a copy activity"
+  workspace_id = var.processing_workspace_id
+  format       = "Default"
+  definition = {
+    "pipeline-content.json" = {
+      source = "${path.module}/pipeline_content.json.tmpl",
+      tokens = {
+          bronze_storage_workspace_id = var.bronze_storage_workspace_id
+          bronze_storage_artifact_id = var.bronze_storage_artifact_id
```

```suggestion
          bronze_lakehouse_id = var.bronze_lakehouse_id
```

Seems more straightforward, wdyt?